application/phoenix/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM elixir:latest
```

Comment:
We should pin the container version here.

Reply:
fixed